config/routes.rb

```diff
@@ -2,6 +2,7 @@
   root 'home_page#landing'
   get 'home_page/index'
   get 'tracks/index'
```

Comment:
Suggestion to change route from ` get 'tracks/index'` to `get 'tracks', to: 'tracks#index`

Before:
<img width="381" alt="Screen Shot 2021-09-18 at 10 41 24 AM" src="https://user-images.githubusercontent.com/78162620/133869723-6a711a39-5bb2-44a9-a841-5f10f4383536.png">

After:
<img width="390" alt="Screen Shot 2021-09-18 at 10 39 53 AM" src="https://user-images.githubusercontent.com/78162620/133869680-6b6b74fd-836b-42ee-a9da-cc434788261c.png">